pages/enterprise/setup.rst

```diff
@@ -57,29 +203,29 @@ When the usage of online repositories is not possible in your environment, you c
 .. note:: These packages can **only** be used when you installed Graylog via the :doc:`/pages/installation/operating_system_packages`!
 
 DEB
-~~~
+---
 
 The installation on distributions like Debian or Ubuntu can be done with *apt-get* as installation tool from the previous installed online repository.
 
 ::
 
-  $ sudo apt-get install graylog-enterprise-plugins
+  $ sudo apt-get install graylog-enterprise-plugins graylog-integrations-plugins graylog-enterprise-integrations-plugins
```

Comment:
did you need to update the parts of the meta package or can you update the meta package?

Reply:
The `graylog-enterprise-plugins` isn't a meta package. If you install it, it'll only install these files:

```
vagrant@instance:~$ dpkg -L graylog-enterprise-plugins
/.
/usr
/usr/share
/usr/share/graylog-server
/usr/share/graylog-server/plugin
/usr/share/graylog-server/plugin/LICENSE-ENTERPRISE
/usr/share/graylog-server/plugin/graylog-plugin-enterprise-4.0.1.jar
/usr/share/graylog-server/bin
/usr/share/graylog-server/bin/chromedriver_start.sh
/usr/share/graylog-server/bin/headless_shell
/usr/share/graylog-server/bin/chromedriver
/usr/share/doc
/usr/share/doc/graylog-enterprise-plugins
/usr/share/doc/graylog-enterprise-plugins/changelog.gz
```

So while installing it does convert the Graylog instance into an Enerprise instance, since we're including the integrations plugins into the enterprise server package, I thought we might as well include them here, too.

Not sure if that's what you meant?

Reply:
Ok, I thought that you could use `graylog-enterprise` as this is a meta package for installation that contains all of the single packages that we have so that you install one package and have 3 installed. 

That is the reason I though we should refer in the docs only to this meta package as main installation option.

Reply:
Oh, I see what you're saying. I think that could work.

Reply:
So then it would delete all the bin files for graylog-server and reinstall them from the `graylog-enterprise` package.

Reply:
Alright, it should be updated now.